pkg/products/threescale/reconciler.go

```diff
@@ -1698,47 +1698,47 @@ func (r *Reconciler) reconcileServiceDiscovery(ctx context.Context, serverClient
 func (r *Reconciler) reconcileBlackboxTargets(ctx context.Context, installation *integreatlyv1alpha1.RHMI, client k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
 	cfg, err := r.ConfigManager.ReadMonitoring()
 	if err != nil {
-		return integreatlyv1alpha1.PhaseInProgress, fmt.Errorf("error reading monitoring config: %w", err)
+		return integreatlyv1alpha1.PhaseInProgress, nil
```

Comment:
We should still return the error message and let what ever consumes the massage decide if it should be logged.

Reply:
Was following this pattern https://github.com/integr8ly/integreatly-operator/blob/master/pkg/products/cloudresources/reconciler.go#L327-L330 
But it seems that There should be a `Log.INFO` to be able to track it